src/NuGet.Clients/NuGet.Console/ScriptExecutor.cs

```diff
@@ -178,7 +187,11 @@ private async Task ExecuteScriptCoreAsync(ScriptExecutionRequest request)
             // to switch to powershell pipeline execution thread. In order not to block the UI thread,
             // go off the UI thread. This is important, since, switches to UI thread,
             // using SwitchToMainThreadAsync will deadlock otherwise
```

Comment:
Does this comment need to be updated? I feel it's doing opposite now.

Reply:
oops, yes, I intended on updating it after I tested the code fix worked, but forgot 🤦‍♂️